src/scripts/components/pages/viewOrders.js

```diff
@@ -3,20 +3,20 @@ import renderToDOM from '../../helpers/renderToDom';
 
 const viewOrders = (array) => {
   clearDom();
-
   let domString = '';
-  domString += `<div class="order-card">
+  array.forEach((obj) => {
+    domString += `<div class="order-card">
   <div class="card-body">
-    <h5 class="card-title">${array.orderName}</h5>
-    <h6 class="card-subtitle mb-2 text-muted">${array.orderStatus}</h6>
-    <p class="card-phone">${array.cusomerPhone}</p>
-    <p class="card-date">${array.customerEmail}</p>
+    <h5 class="card-title">${obj.orderName}</h5>
+    <h6 class="card-subtitle mb-2 text-muted">${obj.orderStatus}</h6>
+    <p class="card-phone">${obj.cusomerPhone}</p>
+    <p class="card-date">${obj.customerEmail}</p>
```

Comment:
Everything looks good. Maybe change the class here to card-email for consistency.